clients/src/test/java/org/apache/kafka/common/network/SelectorTest.java

```diff
@@ -114,10 +114,6 @@ public void testServerDisconnect() throws Exception {
         this.server.closeConnections();
         while (!selector.disconnected().containsKey(node))
             selector.poll(1000L);
-
-        // reconnect and do another request
-        blockingConnect(node);
-        assertEquals("hello", blockingRequest(node, "hello"));
```

Comment:
Can you revert this change please, after setting `closing` flag in `EchoServer#close()`?